CODEOWNERS

```diff
@@ -55,10 +55,10 @@ src/opensuse/15.4/helix/amd64/ @dotnet/dnceng
 
 # raspbian
 src/raspbian/10/crossdeps/arm32v7/ @dotnet/runtime-infrastructure
-src/raspbian/10/helix/arm32v6/ @dotnet/runtime-infrastructure
+src/raspbian/10/helix/arm32v6/ @dotnet/dnceng
 
 # ubuntu
-src/ubuntu/**/helix/ @dotnet/runtime-infrastructure
+src/ubuntu/**/helix/ @dotnet/dnceng

```

Comment:
Thoughts on defining this as a single pattern?

`src/**/helix/ @dotnet/dnceng`

There are actually a few other shared patterns

src/**/cross*/ @dotnet/runtime-infrastructure
src/**/webassembly*/ @dotnet/runtime-infrastructure